src/routes/recover/+page.svelte

```diff
@@ -9,7 +9,7 @@
     let mail: string;
     const recover = async () => {
         try {
-            await sdkForConsole.account.createRecovery(mail, base);
+            await sdkForConsole.account.createRecovery(mail, window.location.hostname);
```

Comment:
I definitely don't think `window.location.hostname` would work because that is just a domain (e.g. `github.com`), but that parameter requires a URL. 

That said, I think we need to make sure we have a [proper recovery page](https://github.com/appwrite/console/issues/160) first.